fuzz/targets/Cargo.toml

```diff
@@ -4,6 +4,9 @@ version = "0.0.1"
 publish = false
 edition = "2018"
 
+[features]
+prost-codec = ["tikv_util/prost-codec"]
```

Comment:
Why `protobuf-codec` not necessary anymore?

Reply:
we only need prost-codec for tikv_util because it optionally depends on prost itself and rust-proto is always included.

But I think fuzz/targets is another root crate and it doesn't depend on tikv? So we need to do something else here?

Reply:
This seems to work actually, `cargo run --package fuzz -- list-targets` compiles and has the expected results.